### PR TITLE
Generator

### PR DESCRIPTION
**Pull request by Jougan-0**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: .github/workflows/meshmodel-generators.yml [hunk 1]
```diff
@@ -74,7 +75,7 @@ jobs:
         with:
           go-version: "1.21"
           
-      - name: Run component Updater
+      - name: Run component Updation
```
Comment (leecalcote) [inline .github/workflows/meshmodel-generators.yml:78]: Please revert.

File: .github/workflows/meshmodel-generators.yml [hunk 2]
```diff
@@ -61,6 +61,7 @@ jobs:
             
   update-components:
     name: Update Components
+    needs: generate-components
```
Comment (leecalcote) [inline .github/workflows/meshmodel-generators.yml:64]: Why was this not previously required?

File: .github/workflows/meshmodel-generators.yml (hunk 1 above)
Comment (leecalcote) [reply .github/workflows/meshmodel-generators.yml:78]:
```suggestion
      - name: Run component Updater

**Comment by saurabh100ni** [comment]
Please add this as your agenda in [Doc](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.69qskjv558bk) to present it in the Website Meet today at 6:30 pm IST/7:00 am CT

**Comment by abhijeetgauravm** [comment]
Hey @Jougan-0  Let's discuss this on website call tomorrow at 8:30 PM IST / 9 AM Central time. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima).

**Review by MUzairS15 — APPROVED** [review]

**Comment by Jougan-0** [comment]
> Hey @Jougan-0 Let's discuss this on Meshery Development Meeting tomorrow at 8:30 PM IST / 9 AM Central time. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit#).

Already merged and required changes have been done already so removing it from the the meeting minutes .

**Comment by leecalcote** [comment]
@Jougan-0 this contains a bug in which a portion of the spreadsheet has been shifted.

**Pull request merged**